.travis.yml

```diff
@@ -0,0 +1,31 @@
+#language: python
+#python:
+#  - "3.6"
+##  - "3.7"
+#services:
+#  - mysql
+#env:
+#  -DJANGO=2.2 DB=mysql
+#install: # => 5
+#  - pip install -r requirements.txt
+#before_script: # => 6
+#  - mysql -e 'create database test;' -u root
+#script: # => 7
+#  - python manage.py test
+#  - pylint --load-plugins pylint_django polls/
+#  - pylint --load-plugins pylint_django posts/
+
+
+services:
+  - docker
+before_install:
+  - docker-compose build
+  - docker-compose up -d
+#before_script: # => 6
+#  - mysql -e 'create database test;' -u root
+script: # => 7
+  - sleep 40
+  - docker exec -t recipeapp_web_1 python manage.py test
+  - docker exec -t recipeapp_web_1 pylint --load-plugins pylint_django recipe/
+  - docker exec -t recipeapp_web_1 pylint --load-plugins pylint_django users/
+  - docker exec -t recipeapp_web_1 pylint --load-plugins pylint_django root/
```

Comment:
Always add a new line at the end of file.

Reply:
Fixed